team/team.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+   </title>
+  </head>
+  <body>
+    <h2>Epic Pi Bots team members</h2>
```

Comment:
The bots must be broken 🤣 

```suggestion
    <h1>Epic Pi Bots team members</h1>
```